src/pages/index.js

```diff
@@ -101,31 +115,33 @@ const CatComponent = () => {
       console.log(`It's a tie between ${winners.join(" and ")}`);
     }
 
-    setPlayers((prevPlayers) => {
-      const resetPlayers = Object.fromEntries(
-        Object.entries(prevPlayers).map(([key, player]) => [
-          key,
-          { ...player, points: 0, selections: [] },
-        ])
-      );
-      return resetPlayers;
-    });
+    // Reset players using the Player class constructor
+    const singlePlayer = new Player("Player");
+    const computer = new Player("Computer");
 
+    setPlayers({ singlePlayer, computer });
+
+    // Reset cats
     setCats((prevCats) =>
-      prevCats.map((cat) => {
+      prevCats.map((cat, index) => {
         const newCat = new Cat(cat.name);
-        newCat.amountOfCheese = cats;
+        newCat.id = index + 1; // Assign unique IDs starting from 1
+        newCat.amountOfCheese = prevCats; // Use the previous cats array to set amount of cheese
```

Comment:
There might be an error here. Please confirm the number of cats that will appear.

Reply:
Ignore this. I think it's fine.